apps/admin/src/components/customFields/SelectApplicant.tsx

```diff
@@ -81,13 +81,13 @@ export const SelectApplicant = ({
 
   useEffect(() => {
     if (members) {
-      // TODO: WHat about pagination?
-      setMemberList(
-        members.map((m) => ({
-          name: m.memberAddress,
-          value: m.memberAddress,
-        }))
-      );
+      // TODO: pagination?
```

Comment:
Why the wait on pagination? Does it work?

Reply:
> Why the wait on pagination? Does it work?

I just edited the comment. i didn't make any other changes, 

adding pagination to this component is not part of this ticket scope.